toolchain/lower/file_context.h

```diff
@@ -111,6 +121,27 @@ class FileContext {
     llvm::SmallVector<SemIR::InstId> param_inst_ids;
     llvm::Type* return_type = nullptr;
     SemIR::InstId return_param_id = SemIR::InstId::None;
+
+    // Compares the types of two functions. This is intended to be used for
+    // specifics of the same generic.
+    auto Equals(FunctionTypeInfo& other) -> bool {
```

Comment:
How does this get used? I was trying to look for calls, but I'm not seeing them. I might not have looked closely for `operator==`, but the special function name caught my eye. Note, holding back on comments since I'm trying to figure out the use.

Reply:
Uses removed in a previous iteration, so removing it as well.